api/src/pcapi/sandboxes/scripts/creators/industrial/create_specific_offers.py

```diff
@@ -0,0 +1,108 @@
+import datetime
+import logging
+
+from pcapi.core.bookings.factories import IndividualBookingFactory
+from pcapi.core.categories import subcategories
+from pcapi.core.offerers.factories import UserOffererFactory
+from pcapi.core.offerers.factories import VenueFactory
+from pcapi.core.offerers.factories import VirtualVenueFactory
+from pcapi.core.offerers.models import Venue
+from pcapi.core.offers.factories import EventOfferFactory
+from pcapi.core.offers.factories import EventStockFactory
+from pcapi.core.offers.factories import ThingOfferFactory
+from pcapi.core.offers.factories import ThingStockFactory
+from pcapi.core.users.factories import BeneficiaryGrant18Factory
+from pcapi.core.users.models import User
+from pcapi.models.offer_mixin import OfferValidationStatus
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_specific_offers() -> None:
+    user_offerer = UserOffererFactory(offerer__name="Offerer avec offres spécifiques")
+    venue = VenueFactory(
+        name="Lieu avec offres spécifiques",
+        managingOfferer=user_offerer.offerer,
+    )
+    # offerers have always a virtual venue so we have to create one to match reality
+    VirtualVenueFactory(
+        name="Lieu virtuel avec offres spécifiques", managingOfferer=user_offerer.offerer, isVirtual=True
```

Comment:
(détail) `isVirtual` est déjà bien défini par `VirtualVenueFactory`

```suggestion
        name="Lieu virtuel avec offres spécifiques", managingOfferer=user_offerer.offerer
```